frontend/src/components/common/EditModuleModal.tsx

```diff
@@ -54,17 +66,51 @@ const EditModuleModal = ({
       );
       return;
     }
-    const newModule = { ...module, title, description, published: isPublished };
-    const [id, course] = formatCourseRequest(newModule);
-    updateCourse({
-      variables: { id, course },
-      refetchQueries: [
-        { query: COURSES },
-        { query: GET_COURSE, variables: { id } },
-      ],
-    });
-    setInvalid(false);
-    onClose();
+
+    if (previewImage) {
```

Comment:
Probably best to skip this check, I don't think images are required for modules.